packages/flutter/lib/src/widgets/scroll_view.dart

```diff
@@ -438,6 +438,9 @@ class ListView extends BoxScrollView {
   /// children because constructing the [List] requires doing work for every
   /// child that could possibly be displayed in the list view instead of just
   /// those children that are actually visible.
+  ///
+  /// It is usually more efficient to create widgets on demand using [new
```

Comment:
The previous paragraph refers to "widgets" as children, maybe say children here as well.